### PR TITLE
Prototype faq

### PR DESCRIPTION
**Pull request by sofer**
Updated copy for prototyping service.

### CODE REVIEW

File: _harp/agency/services/prototyping.md [hunk 1]
```diff
@@ -1,86 +1,50 @@
...
+*What happens if I want further work done on my prototype?*  
+We encourage you to retain one or more of the development team to continue to work on your project on a freelance or permanent basis after delivery of the prototype.
+
+*Who owns the IP on my prototype?*  
+You will own the intellectual property. Your development team will handover all the software, documented and tested, on completion of the project.
+
+*What is the advantage of this service?*  
+The great advantage of this service is that it will give you an opportunity to work with some very talented developers early in their careers. If you are looking to hire developers or build a team for a new project, you are unlikely to find a better source of affordable talent in London.
+
+*What if I am past the MVP or prototype stage and am ready to build a scalable, fully-functioning application?*
```
Comment (iteles) [inline _harp/agency/services/prototyping.md]: I'm not entirely sure this is accurate as we talk about MVPs at every client meeting and I would like to expand on our relationship, but we can iterate on this :) :+1:

**Comment by rub1e** [comment]
I also think we can be a bit more inspirational in the way we sell ourselves here, and I think I prefer it  if we link to a dedicated MVP form rather than just the contact page, but as @iteles says we can iterate - a good time for this will be when we update the whole site. Soon.

**Comment by sofer** [comment]
@rub1e If we are going to do an MVP form, I want something really nice, not some crappy google form.

File: _harp/agency/services/prototyping.md (hunk 1 above)
Comment (sofer) [inline _harp/agency/services/prototyping.md]: @iteles Please do suggest a copy amendment when you've thought of something more appropriate (it doesn't have to await a wholesale redesign).

**Comment by rub1e** [comment]
@sofer totally agree - and I think it's worth devoting a bit of thought to it because it's a mildly daunting thing for a client to be presented with a blank contact form - much nicer to guide them through the info we need from them

**Pull request merged**